tests/unit-tests/python/foglamp_test/services/core/test_scheduler.py

```diff
@@ -26,9 +26,9 @@
 # TODO: To run this test,
 #       1) Do 'foglamp start' and note the management_port from syslog
 #       2) Change _m_port below with the management_port
-#       3) Execute this command: FOGLAMP_ENV=TEST pytest -s -vv tests/core/test_scheduler.py
+#       3) Execute this command: FOGLAMP_ENV=TEST pytest -s -vv tests/unit-tests/python/foglamp_test/services/core/test_scheduler.py
```

Comment:
Approach is yet to be decided to eliminate any manual steps

Reply:
Yes, add a TODO. but let's atleast have the ability to run these automated tests with manual intervention; 

i.e.find and set core mgt port, start/ stop foglamp etc.